openml/exceptions.py

```diff
@@ -14,3 +14,10 @@ class OpenMLCacheException(PyOpenMLError):
     """Dataset / task etc not found in cache"""
     def __init__(self, message):
         super(OpenMLCacheException, self).__init__(message)
+
+
+class OpenMLRestrictionViolated(PyOpenMLError):
```

Comment:
In a recent change, Jan implemented a string-length check on the server, so this is no longer necessary. I removed the problematic code.